packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -848,3 +930,109 @@ describe('ripgrep-search-in-workspace-server', function (): void {
         }
     });
 });
+
+describe('ripgrep-search-in-workspace-server-matchPatternToPathResult', function (): void {
+    this.timeout(10000);
+    it('Resolve pattern to path for filename', function (): void {
```

Comment:
We should follow the format of existing tests:

```suggestion
    it('should resolve pattern to path for filename', function (): void {
```

The same is true for all other tests.

Reply:
Done